styles/login.module.css

```diff
@@ -0,0 +1,84 @@
+.segment {
+  margin: 50px 50px 0px 50px !important;
+    width: 600px !important;
+  padding: 50px 75px !important;
+}
+@media only screen and (max-width: 620px) {
+  .segment {
+    padding: 10px !important;
+    }
+    }
+
+    @media only screen and (max-width: 527px) {
+      .toOppositeSides {
+        flex-direction: column;
+        align-items: flex-start !important;
+      }
+
+      .forgotPasswordBreadcrumb {
+        margin-top: 10px;
+      }
+
+      .signUpNowBreadcrumb {
+        justify-content: flex-start !important;
+      }
+
+      .questionHeader {
+        display: flex;
+        justify-content: flex-start !important;
+        display: flex;
+        justify-content: flex-start;
+  }
+}
+
+.loginHeader {
+  font-family: var(--mainFont) !important;
+}
+
+.loginButton {
+  font-family: var(--mainFont) !important;
+  height: 38px !important;
+}
+
+.rememberMeButton {
+  font-family: var(--mainFont);
+}
+
+.forgotPasswordBreadcrumb {
+  font-family: var(--mainFont);
+}
+
+.questionHeader {
+  font-family: var(--mainFont) !important;
+}
+.signUpNowBreadcrumb {
+  font-family: var(--mainFont);
+}
+
+.centerMain {
+  display: flex;
+  justify-content: center;
+}
+.center {
+  display: flex;
+  justify-content: center;
+    align-items: center;
+  }
+
+    .toOppositeSides {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
```

Comment:
It might also be a good idea to include line 75 here as well. However, I think it makes sense to keep ```.verticalSpacing```separate because you use it more than once and helps maintain organization and reusability. What do you think?

Reply:
I think if it were the same case as the redundant lines of code only used once, then it would be good to include line 75. But I reuse vertical spacing multiple times and adding line 75 would perform two functions at once, a combination I only use once. So I think it would be best to keep .verticalSpacing separate, at least in this case.